test/macros.jl

```diff
@@ -161,6 +161,15 @@ let (err, st) = try
     @test any(sf->sf.func===Symbol("@m_throw"), st)
 end
 
+let res = try
+        JuliaLowering.include_string(test_mod, "_never_exist = @m_not_exist 42")
```

Comment:
Let's move this to the IR tests instead. Errors in the IR tests are formatted nicely, including the source range.  For examples, see here https://github.com/c42f/JuliaLowering.jl/blob/cfa6901d597dc27a303c28cc87fed608eb76df2a/test/macros_ir.jl#L93

Reply:
Added the IR test. I would still like to leave this test case in macros.jl since I will add this new test case checking in #27: https://github.com/c42f/JuliaLowering.jl/blob/6407861d744f49737ddc43e7c2f2aaca92b93e89/test/macros.jl#L172